python/ray/tests/test_basic.py

```diff
@@ -2840,3 +2840,23 @@ def test_shutdown_disconnect_global_state():
     with pytest.raises(Exception) as e:
         ray.global_state.object_table()
     assert str(e.value).endswith("ray.init has been called.")
+
+
+def test_redis_lru_with_set(shutdown_only):
+    ray.init(object_store_memory=10**8)
```

Comment:
Do you need use many cpus? You can use `ray_start_regular` according to https://github.com/ray-project/ray/pull/4390 . Now the test fixtures is refactors and we want the user to avoid using `shutdown_only`, because there is some default test setting can be applied. `object_store_memory=10**8` is one of the default setting.

Reply:
Replaced `shutdown_only` with `ray_start_object_store_memory`